app/views/wpcc/your_results/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <section class="section section--contributions">
   <h2 class="section__heading contributions__heading">2. <%= t('wpcc.contributions.title') %>
-    <span class="section__heading-summary">You: <%= session[:employee_percent] %>%, Your employer: <%= session[:employer_percent] %>%</span>
+    <span class="section__heading-summary"><%= t('wpcc.results.contribution_changes.you') %>: <%= session[:employee_percent] %>%, <%= t('wpcc.results.contribution_changes.employer') %>: <%= session[:employer_percent] %>%</span>
```

Comment:
can some attempt be made to keep to a limit of the desired 80 char limit by formatting this line over multiple lines e.g.
```
<span class="section__heading-summary">
  <%= t('wpcc.results.contribution_changes.you') %>: 
  <%= session[:employee_percent] %>%, 
  <%= t('wpcc.results.contribution_changes.employer') %>: <%= session[:employer_percent] %>%
</span>
```